cmd/stolon-pgbouncer/main.go

```diff
@@ -55,10 +55,11 @@ var (
 	childProcessTerminationGracePeriod  = supervise.Flag("termination-grace-period", "Pause before rejecting new PgBouncer connections (on shutdown)").Default("5s").Duration()
 	childProcessTerminationPollInterval = supervise.Flag("termination-poll-interval", "Poll PgBouncer for outstanding connections at this rate").Default("10s").Duration()
 
-	pauser                 = app.Command("pauser", "Serve the PgBouncer pause API")
-	pauserPgBouncerOptions = newPgBouncerOptions(pauser)
-	pauserToken            = pauser.Flag("token", "Authentication token for pauser API").Default("").Envar("STBOUNCER_FAILOVER_TOKEN").String()
-	pauserBindAddress      = pauser.Flag("bind-address", "Listen address for the pauser API").Default(":8080").String()
+	pauser                     = app.Command("pauser", "Serve the PgBouncer pause API")
+	pauserPgBouncerOptions     = newPgBouncerOptions(pauser)
+	pauserToken                = pauser.Flag("token", "Authentication token for pauser API").Default("").Envar("STBOUNCER_FAILOVER_TOKEN").String()
+	pauserBindAddress          = pauser.Flag("bind-address", "Listen address for the pauser API").Default(":8080").String()
+	pauserInitialResumeTimeout = pauser.Flag("initial-resume-timeout", "Timeout for initially resuming PgBouncer on start-up").Default("5s").Duration()
```

Comment:
Does this need to be separate from our other timeout configs? Is there a sensible one we could re-use (no idea if there is, just wondering)?

Reply:
We don't have any unfortunately. We have a failover flag that is `failoverResumeTimeout` but it's available on a different command. Figured it made sense to be very specific about what this flag does, given it doesn't harm us and we may one day decide waiting this long before booting is a bad idea.